tools/clang/test/HLSLFileCheck/samples/d3d11/TessellatorCS40_TessellateIndicesCS.hlsl

```diff
@@ -581,6 +581,7 @@ void main( uint3 DTid : SV_DispatchThreadID, uint3 Gid : SV_GroupID, uint GI : S
                 else
                 {
                     int ring = GetRingFromIndexStitchRegular(true, DIAGONALS_MIRRORED, processedTessFactors.numPointsForOutsideInside.w, index_id - num.z);
+                    DebugOutput[DTid.x+1] = int4(ring,0,0,0); // HLSL Change: Use that output so the value defines it doens't get deleted.
```

Comment:
I don't understand the need for this added line.

Reply:
The test was checking for a Sqrt in this function, but with this new change, this calls turns out to be not needed, so i'm artificially creating a use for the call so it sticks around.

Reply:
Undid this change. Turns out it was broken by an earlier iteration, which was incorrect.